src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/AuthenticationController.kt

```diff
@@ -50,4 +50,11 @@ class AuthenticationController(context: ActionContext,
         context.addResponseHeader("Access-Control-Allow-Credentials", "true")
         return okayResponse()
     }
+
+    fun clearShinyCookie(): String
+    {
+        context.addResponseHeader("Set-Cookie", "jwt_token=; Path=/; Secure; HttpOnly; SameSite=Lax")
```

Comment:
Maybe these lines could be factored out to be shared with `setShinyCookie`, and just pass an empty token string for the clear case?